tests/unit/test_zigzag.py

```diff
@@ -200,6 +202,7 @@ def test_exceeds_max_file_size(self, flat_all_passing_xml, simple_json_config, m
             zz = ZigZag(flat_all_passing_xml, simple_json_config, TOKEN)
             zz.parse()
 
+    @pytest.mark.skip(reason="This seems like a dupe of a test I fixed in the CLI tests")
```

Comment:
Is there a tech debt ticket to revisit this later?

Reply:
https://rpc-openstack.atlassian.net/browse/ASC-1558